app/src/main/res/values-bg/strings.xml

```diff
@@ -17,8 +17,8 @@
 <!-- smartling.entity_escaping = false -->
 <!-- smartling.instruction_attributes = instruction -->
 <resources xmlns:tools="http://schemas.android.com/tools">
-    <string name="appName" translatable="false">DuckDuckGo</string>
-    <string name="appDescription" translatable="false">DuckDuckGo</string>
```

Comment:
Not sure why we removed this?

Reply:
Good catch! This change came from Smartling, so it's not related to my changes. To fix it, I'll merge my branch first and then create a new Smartling job in a different branch so it gets updated remotely in Smartling as well.